### ISSUE TITLE
FastForestOva has its prediction value in Score[1] instead of Score[0]

### ISSUE THREAD

**Issue by andrasfuchs** [dotnet/machinelearning#6037]
**System Information (please complete the following information):**
 - OS & Version: Windows 11
 - ML.NET Version: ML.NET v1.7
 - .NET Version: .NET 6.0
 - ML.NET Model Builder: ML.NET Model Builder 2022 v16.9.2.2205603

**Describe the bug**
Usually the probabilities of the trained models are stored in the Score values of the `ModelOutput` class (generated by ML.NET Model Builder). The first item in that array of floats is the prediction value when we use the `FastTreeOva` or `LightGbmMulti` algorithms, but interestingly it is the second item in that array if we use `FastForestOva`.
I'm not 100% sure that it is a bug, but I would expect consistency in this regard among the different algorithms.

**To Reproduce**
1. Go to Solution Explorer
2. Right click on the project, add Machine Learning Model
3. Start training your model, and stop at the different times to have different algorithms as best
4. Test the generated classes by running them with your input data and show the ModelOutput.Score array
5. Observe the different values and their meaning, especially for the `FastForestOva` algorithm

**Expected behavior**
I would expect all the algorithms to return their prediction values in the same order. 

**Screenshots, Code, Sample Projects**
Accuracy is calculated based on `OutputModel.Score[0]` values and `IsAttached` uses `LightGbmMulti`, `IsNotAttached` uses `FastForestOva`. They both had 99+% accuracy during training.
![image](https://user-images.githubusercontent.com/910321/148792403-a89a6af2-e3c8-4534-b3ce-3b80c05dd83e.png)

**Additional context**
I could test `FastTreeOva`, `LightGbmMulti` and `FastForestOva`: the first two had their prediction value in Score[0], and `FastForestOva` had its in Score[1].

**Comment by michaelgsharp** [comment]
I would agree that consistency is important... we will have to see what is causing this.

@JakeRadMSFT is this possibly due to how model builder is generating the Model Output file? Or do we need to go dig through the ML.NET code and see whats going on?

**Comment by michaelgsharp** [comment]
@andrasfuchs would you be able to share your training pipeline? Both with this one and #6035?

**Comment by andrasfuchs** [comment]
Sure, I use Visual Studio 2022 v17.0.4 with the ML.NET Model Builder 2022 v16.9.2.2205603 extension. I have a C# console application that includes these mbconfigs to generate the ML models: 
https://drive.google.com/file/d/1CZJitEjEd3GEhBbHZUMOgEbRg3GaPsMD/view?usp=sharing

You can find the data that I used for training and for the model accuracy measurements:
https://drive.google.com/file/d/1C2Of2UIHN2y7l5J-SvEvacFhTAHMCCcm/view?usp=sharing

My source code is work in progress, but you can check it out here:
https://github.com/andrasfuchs/BioBalanceDetector/tree/master/Software/Source/BBDProto08/BBD.SleepLogger

Let me know if you need anything more!

**Comment by andrasfuchs** [comment]
Hi guys, did you have the chance to look into this? Do you need any more data from me to reproduce the problem?

Until then, is there any workaround for this issue, can I remove some of the training algorithms from the training loop of the model builder?
Or is there a way to manually choose the algorithm for the code/model generator in model builder?
